### PR TITLE
Parent PLUS loans separated from other family contributions

### PR DESCRIPTION
**Pull request by mistergone**
Parent PLUS loans are now a separate field!
## Changes
- Parent PLUS added as a field
  - "Money your family or others will pay" no longer includes Parent PLUS loans
  - "Money your family will provide" in the sidebar is now the total of family contributions and Parent PLUS loans (which was true before, but now is explicitly added together to form a new property, `familyTotal` specifically to fill this field)
- The `student-debt-calc` package was updated to `v2.4.1`, which allows Parent PLUS loans to be added properly and fixes the incorrect calculation of federal loan fees.
- Functional test values were updated to consider the following: 
  - The removal of federal loan fees from the principal
  - The separation of Parent PLUS loans from family contributions
  - **Note:** New values were checked using a debt calculation spreadsheet.
## Testing
- Unit tests passing
- Functional tests mostly passing
  - 7 failures relate to budget worksheet changes that require a fix
## Review
- Pull it down, `npm install` and `gulp` and try out that new Parent PLUS loan field!
- @marteki @higs4281 (and, if bored, @niqjohnson )
## Screenshots

screen shot 2016-05-25 at 1 34 33 pm
## Checklist
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code can be automatically merged (no conflicts)
- [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
- [x] Passes all existing automated tests
- [x] New functions include new tests
- [x] New functions are documented (with a description, list of inputs, and expected output)
- [x] Placeholder code is flagged
- [x] Visually tested in supported browsers and devices 
- [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)
## Animated GIF

t-rex-dance

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,6 +61,6 @@
     "number-to-words": "1.2.2",
     "require-dir": "0.3.0",
     "snyk": "1.14.3",
-    "student-debt-calc": "2.3.0"
+    "student-debt-calc": "^2.4.1"
```
Comment (marteki) [inline package.json]: Please remove the carat to pin this dependency, then re-shrinkwrap the packages. :-)

Comment (mistergone) [inline package.json]: airplane-unplugged

**Comment by marteki** [comment]
Using a URL with a `ppl` value of $1000:

screen shot 2016-05-25 at 3 43 36 pm

And with it set to $0:

screen shot 2016-05-25 at 3 43 48 pm

:+1:

**Comment by mistergone** [comment]
Merging!

merge6

**Pull request merged**